### DISCUSSION TITLE
I wanted to customize my tools, but I found I couldn't import them into my work scene

### DISCUSSION THREAD

**Discussion by chatPower** [FlowiseAI/Flowise#906] [General]
I wanted to customize my tools, but I found I couldn't import them into my work scene

**Comment by chungyau97** [comment]
Hi @chatPower, 

Troubleshooting steps:
1. `Marketplaces` > `Tools`
2. Click `Send Discord Message` 
![image](https://github.com/FlowiseAI/Flowise/assets/33013947/4df4883c-ef6e-4457-a7f9-667715c31c75)

3. Click `Use Template`
4. Click `Add`
5. Navigate to `Tools` 
6. Click `send_message_to_discord_channel`
7. Click `Export`
![image](https://github.com/FlowiseAI/Flowise/assets/33013947/4ad227b9-6402-43bf-8720-f741038b6f4f)
8. Click `Load` and select the exported file that is in JSON format
